pkg/api/user.go

```diff
@@ -434,8 +434,8 @@ func (hs *HTTPServer) ChangeUserPassword(c *contextmodel.ReqContext) response.Re
 	}
 
 	getAuthQuery := login.GetAuthInfoQuery{UserId: usr.ID}
-	if err := hs.authInfoService.GetAuthInfo(c.Req.Context(), &getAuthQuery); err == nil {
-		authModule := getAuthQuery.Result.AuthModule
+	if authInfo, err := hs.authInfoService.GetAuthInfo(c.Req.Context(), &getAuthQuery); err == nil {
```

Comment:
Unrelated thought, I'm wondering why the error isn't checked if it is non-nil. Please disregard this comment unless you know the answer off the top of your head. It's not about the change being made in this PR.